amundsen_application/static/js/components/common/Flag/index.tsx

```diff
@@ -41,7 +41,9 @@ const Flag: React.SFC<FlagProps> = ({
   // https://getbootstrap.com/docs/4.1/components/badge/
   return (
     <span className={`flag label label-${labelStyle}`}>
+      <div className={`badge-overlay-${labelStyle}`}>
```

Comment:
Curious, if you use a <span> instead, would you be able to avoid setting `display: inline-block` in the styles above?

Reply:
You mean `a` as in a hyperlink tag?

Reply:
I actually don't think it is needed either way, testing it out when I remove it nothing changes so might just have been leftover from my experimenting with different styles

Reply:
Oh, something weird happened with the markdown, I meant a '\<span\>'